src/Persistent/Pure.hs

```diff
@@ -425,6 +427,23 @@ findUser = findInById dbUserMap
 getUsers :: Query [User]
 getUsers = view dbUsers
 
+getUsersInClass :: SchoolClass -> Query [User]
+getUsersInClass clss = filter (isClassInRole clss . view userRole) <$> view dbUsers
```

Comment:
you can move this one there if you want, but you'll have to do it yourself.  (-: